README.md

```diff
@@ -0,0 +1,74 @@
+# 2D Simulator
+
+## Quick start
+
+To run the simulator, use the ``makeSim`` command, for example:
+
+    makeSim --visit 35 --detector b1 --pfsDesignId 3 --exptime 30 --type flat --imagetyp flat_odd
+
+The above example generates a 30-second simulated flat exposure corresponding to visit 35,
+for detector ``b1``, and for odd-numbered fibers, with the active fibers provided
+in pfsDesign ID 3.
+
+    makeSim --visit 58 --detector m1 --pfsConfig --pfsDesignId 1 --exptime 300 --type object --objSpectraDir /projects/HSC/PFS/simulator/
+
+The next example above a 300-second simulated object exposure corresponding to visit 58,
+for detector ``m1``, with the objects associated with each active fiber listed
+in pfsDesign ID 1. Note in the case of object exposures, object spectra are required.
+Their location are provided by the ``objSpectraDir`` argument. In this case, `/projects/HSC/PFS/Simulator`.
+
+See the simulator [User Guide](https://sumire.pbworks.com/w/file/fetch/131637714/PFS-DRP-PRU060011-01_PFS2DDRPSimulatorUserDocumentation.pdf) for more information regarding usage.
+
+## Object Spectra catalogs
+
+For running science object simulations (ie where the ``--type`` is ``object``), object spectra are required,
+in the form of `pfsSimObject` FITS files. Each object spectrum (`pfsSimObject`) file is identified by the
+catalog ID or `catId`, and the object identifier for the object within that catalog `objId`.
+The object spectra files needed by the simulator must be organized under a single root directory, which
+is the value of the ``--objSpectraDir`` argument.
+
+This root directory must contain a file ``catalog_config.yaml``, which specifies the available catalogs,
+their unique ``catId`` s, and their location relative to the root directory.
+Specifically, the YAML file must contain a sequence of one or more mapping nodes,
+where each mapping node consists of three key-value pairs: the `catId`, the catalog `name`, and
+the `rel_location` (relative location).
+
+Below is an example of a
+``catalog_config.yaml`` file that is used by the Princeton group:
+
+```
+- catId: 1
+  name: lowz_COSMOS
+  rel_location: pfsSimObjects/1
+- catId: 5
+  name: GA_Laszlo_2021-04
+  rel_location: pfsSimObjects/5
+- catId: 6
+  name: GE_combined
+  rel_location: pfsSimObjects/6
```

Comment:
Why name the directories with just a number???
We need a mapping from `catId` to directory name, so that the directory name can be meaningful.

Reply:
Added directories with descriptive names.